Update name of Apollo Runtime Container throughout (#152)
apollographql/apollo-mcp-server: docs/source/command-reference.mdx

</ExperimentalFeature>

## Installation Methods

### Download container image

A container is built for the Apollo MCP Server with every release at `ghcr.io/apollographql/apollo-mcp-server`.

> To download the **latest release** Docker container of Apollo MCP Server:

Reviewer: Are there non-Docker ways to do this? Also is this our recommended method? I pulled it up because we started talking about the container.

Reviewer 2: There are Docker alternatives with their own pull mechanisms, but providing the Docker commands is probably sufficient.

</Note>

### Apollo Runtime Container

> The Apollo Runtime Container runs both the MCP Server and the [Apollo Router]((https://www.apollographql.com/docs/graphos/routing in a single container. It's useful for local development, testing, and production deployments.

Reviewer 3: ```suggestion
The Apollo Runtime Container runs both the MCP Server and the [Apollo Router](https://www.apollographql.com/docs/graphos/routing) in a single container. It's useful for local development, testing, and production deployments.
```

Reviewer: Oops thanks for catching this!

> The Apollo Runtime Container enables you to run both the MCP Server and the Apollo Router, all in a single container. This is useful for local development and testing, as well as for production deployments where you want to run both services together.

Reviewer: ```suggestion
The Apollo Runtime Container runs both the MCP Server and the [Apollo Router]((https://www.apollographql.com/docs/graphos/routing in a single container. It's useful for local development, testing, and production deployments.
```

Using Apollo Librarian suggestion and adding a link
